frontend/src/routes/LandingPage/Admission.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import djangoData from "src/utils/djangoData";
+import LegoButton from "src/components/LegoButton";
+import { media } from "src/styles/mediaQueries";
+import FormatTime from "src/components/Time/FormatTime";
+import { useMyApplication } from "src/query/hooks";
+import { Admission as AdmissionInterface } from "src/types";
+import CountDown from "./CountDown";
+
+interface AdmissionProps {
+  admission: AdmissionInterface;
+}
+
+const Admission: React.FC<AdmissionProps> = ({ admission }) => {
+  const [hasSubmitted, setHasSubmitted] = useState(false);
+
+  const { data: myApplication } = useMyApplication(admission.pk);
+
+  useEffect(() => {
+    setHasSubmitted(!!myApplication);
+  }, [myApplication]);
+
+  return (
+    <AdmissionWrapper>
+      <AdmissionTitle>{admission.title}</AdmissionTitle>
+      <AdmissionDescription>
+        {admission.description.split("\n").map((descriptionLine, index) => (
+          <React.Fragment key={index}>
+            {descriptionLine}
+            <br />
+          </React.Fragment>
+        ))}
+      </AdmissionDescription>
+
+      <AdmissionDetails>
+        <TimeLineWrapper>
+          {!admission.is_open && (
+            <TimeLineItem
+              title="Opptaket åpner"
+              dateString={admission.open_from}
+              details={[]}
+            />
+          )}
+          <TimeLineItem
+            title="Søknadsfrist"
+            dateString={admission.public_deadline}
+            details={[
+              "Alle søknader innsendt innen søknadsfristen er garantert å bli behandlet.",
```

Comment:
A bit long sentence maybe? Is it still understandable with "Blir garantert behandlet"?

Reply:
I'm considering scrapping that entire sentence, as I feel like it's implicit in "Søknadsfrist". Plus with the design as it is now it is specified that it's not guaranteed that your application will be considered before the editing deadline - which sortof implies that it will be considered if you apply earlier.